content/docs/hooks-faq.md

```diff
@@ -820,7 +820,7 @@ function Image(props) {
 }
 ```
 
-`useRef` **不**接受像 `useState` 這樣的特殊重載函式。你可以撰寫你自己的函式來建立並延遲設定：
+`useRef` **不**接受像 `useState` 這樣特殊的重載 function 。你可以撰寫你自己的 function 來建立並延遲設定：
```

Comment:
```suggestion
`useRef` **不**接受像 `useState` 這樣特殊的重載 function。你可以撰寫你自己的 function 來建立並延遲設定：
```

Reply:
Haha I know, thank you coming back Taiwan for voting!